drivers/ksfilter/ks/swenum.c

```diff
@@ -2109,7 +2115,16 @@ KsServiceBusEnumPnpRequest(
 
             /* complete pending irps */
             KspCompletePendingIrps(DeviceEntry, STATUS_DEVICE_REMOVED);
-
+            
+			/* free device extension */
```

Comment:
fix the wrong indentation level of that comment.

Reply:
Fixed.